librz/analysis/analysis.c

```diff
@@ -103,8 +103,8 @@ RZ_API RzAnalysis *rz_analysis_new(void) {
 	analysis->diff_thbb = RZ_ANALYSIS_THRESHOLDBB;
 	analysis->diff_thfcn = RZ_ANALYSIS_THRESHOLDFCN;
 	analysis->syscall = rz_syscall_new();
-	analysis->arch_target = rz_arch_target_new();
-	analysis->platform_target = rz_arch_platform_target_new();
+	analysis->arch_target = rz_platform_target_new();
```

Comment:
Should member also bee renamed?

Reply:
i think it won't matter once we merge RzAnalysis with RzAsm